run.sh

```diff
@@ -97,9 +97,12 @@ podman run \
     --mount type=bind,source="$(dirname $0)/conf",destination=/opt/cryostat.d/conf.d,relabel=shared \
     --mount type=bind,source="$(dirname $0)/templates",destination=/opt/cryostat.d/templates.d,relabel=shared \
     --mount type=bind,source="$(dirname $0)/truststore",destination=/truststore,relabel=shared \
+    --mount type=tmpfs,target=/opt/cryostat.d/probes.d \
     -e CRYOSTAT_PLATFORM=$CRYOSTAT_PLATFORM \
     -e CRYOSTAT_DISABLE_SSL=$CRYOSTAT_DISABLE_SSL \
     -e CRYOSTAT_DISABLE_JMX_AUTH=$CRYOSTAT_DISABLE_JMX_AUTH \
+    -e CRYOSTAT_DISABLE_SSL="false" \
+    -e CRYOSTAT_DISABLE_JMX_AUTH="false" \
```

Comment:
Can these two lines be removed?

Reply:
yes, these were for local testing, I've removed them now